packages/oid4vci-demo-frontend/src/configs/energy_shr.json

```diff
@@ -1,7 +1,7 @@
 {
   "general": {
-    "credentialName": "Conference Attendee",
-    "issueCredentialType": "dbc2023",
+    "credentialName": "EnergySHR Guest",
+    "issueCredentialType": "energySHR",
     "oid4vpAgentBaseUrl": "https://ssi.dutchblockchaincoalition.org/demo/issuer",
```

Comment:
Although this demo is currently not deployed, and although it is not part of this PR, please update all the ssi.dutchblockchaincoalition.org to ssi.energyshr.nl for the energyshr config(s)

Reply:
changed to https://www.energyshr.nl/